packages/salesforcedx-vscode-replay-debugger/package.json

```diff
@@ -89,6 +104,12 @@
             "view == sfdx.force.view.checkpoint && sfdx:project_opened && sfdx:replay_debugger_checkpoints_enabled",
           "group": "navigation"
         }
+      ],
+      "editor/context": [
+        {
+          "command": "sfdx.launch.replay.debugger.logfile",
+          "when": "editorLangId == 'apexlog'"
```

Comment:
Should we also check if this is a sfdx project? I am not sure if we can hook into that context from the salesforcedx-core extension. Maybe sfdx_project_opened is universal and applicable.

Could you check?

Reply:
Seems like `sfdx:project_opened` is universal. The checkpoints view uses that context as well.